nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -78,7 +78,7 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & state)
       500), false);
 
   from_ll_to_map_client_ = node->create_client<robot_localization::srv::FromLL>(
-    "/fromLL",
+    "fromLL",
```

Comment:
The problem with this is that then the service will be placed under a namespace that the waypoint follower node is placed in -- not the namespace that potentially the robot localization node is. I think this may be better to be remapped in your launch file from `/fromLL` -> `/my_rl_ns/fromLL` that way you explicitly control what namespace this is applied to. There's no reason to expect that the namespace of the WPF will be the same as RL - especially if you consider that many GPS applications may have 2 or 3 RL instances that could either be named uniquely or namespaced uniquely - separate of the application namespace itself.